### ISSUE TITLE
Skate keyboard controls are broken when VoiceOver is enabled

### ISSUE THREAD

**Issue by samreid** [phetsims/energy-skate-park#589]
If you hold down the up arrow and while that is held down, press the left arrow, the skater motion gets confused, but only when VoiceOver is on.

Steps:
1. Launch the sim.
2. Enable VoiceOver
3. Tab to the skater
4. Press spacebar to grab the skater
5. Press and hold the up arrow
6. tap and release the left arrow (still holding the up arrow)
7. release the up arrow

After doing this, the user is pressing no keys yet the skater keeps flying toward the top of the screen. This is not a problem with voiceover off. @jessegreenberg any ideas?

**Comment by samreid** [comment]
In https://github.com/phetsims/energy-skate-park/issues/581 I saw this affects other movable things like track points and the energy sensor probe. This seems like a very severe problem.

**Comment by jessegreenberg** [comment]
VoiceOver has a bug where keyup events are not emitted when multiple keys are pressed at once. But I thought we had a scenery workaround in place exactly for this, I will take a look. 

Sorry about this! It is almost certainly not a sim specific problem.

**Comment by jessegreenberg** [comment]
The workaround was introduced in https://github.com/phetsims/balloons-and-static-electricity/issues/384 and dates all the way back to 2018. It is Safari specific since we cannot detect whether VoiceOver is active.

Current code: https://github.com/phetsims/totality/blob/7e398eeb36e6c632f62eb1a08d19ad1bb6266e32/scenery/js/listeners/KeyboardDragListener.ts#L759-L765

On Safari, only one movement key can be pressed at a time. The original rationale was that PhET officially supports VoiceOver on Safari, so limiting simultaneous movement keys there addressed the VoiceOver + Safari interaction without affecting other browsers.

I made https://github.com/phetsims/scenery/issues/1787 to see if there is a better way today. I don't have any ideas, but our more modern code and tools might offer a better way.

For now, I don't think this should block this release, since this impacts all sims similarly. @samreid are you OK with this plan?

**Comment by samreid** [comment]
Thanks, I had forgotten about it. It seems good to continue and thanks for opening https://github.com/phetsims/scenery/issues/1787
